content/rs/security/tls-ssl.md

```diff
@@ -32,6 +32,10 @@ You can enable TLS by editing the configuration of an existing database (as show
     - If your database uses Replica Of or Active-Active replication, you will need to add the syncer certificates for the participating clusters. The steps for each are below.
 1. Optional: To limit connections further to a subset of those with valid certificates, enforce **Subject Alternative Name** and enter authorized users separated with commas.
 1. Select **Update** at the bottom of the screen to save your configuration.
+1. Optional: Redis Enterprise enforce client certificate expiration date validation by default. It can be disabled on the database level using rladmin tool.
```

Comment:
```suggestion
1. Optional: Redis Enterprise enforces client certificate expiration date validation by default. It can be disabled on the database level using rladmin tool.
```